app/assets/javascripts/spotlight/croppable.js

```diff
@@ -47,38 +47,48 @@ Spotlight.onLoad(function() {
         $('#' + cropid + '_crop_h').val(coords.h);
         updatePreview(coords);
       };
-      cropbox.Jcrop(
 
-        $.extend(options, {
-          onSelect: update,
-          onChange: update
-        })
-      );
+      if(!cropbox.data('jcropProcessed')){
+        cropbox.Jcrop(
+          $.extend(options, {
+            onSelect: update,
+            onChange: update
+          })
+        );
+      }
+      cropbox.data('jcropProcessed', 'true');
 
       fileUpload.on('change', function() {
-
-        var file = this.files[0];
-        var img = cropbox[0];
-
-        img.file = file;
-
-        var reader = new FileReader();
-
-        reader.onload = (function(aImg) { return function(e) {
-          var jcrop_api = cropbox.data('Jcrop');
-          jcrop_api.setImage(e.target.result);
+        var jcrop_api = cropbox.data('Jcrop');
+        if(this.files){
+          var file = this.files[0];
+          var img = cropbox[0];
+
+          img.file = file;
+
+          var reader = new FileReader();
+
+          reader.onload = (function(aImg) { return function(e) {
+            jcrop_api.setImage(e.target.result);
+            cropbox.css({width: "", height: ""});
+            cropbox[0].src = e.target.result;
+            if(previewbox.length > 0) {
+              previewbox[0].src = e.target.result;
+            }
+            jcrop_api.setSelect([0,0,200,200]);
```

Comment:
I'm not sure it's worth tackling now, but we should probably make this default selection configurable. I think this size came from the contacts' avatar crop, right?